blocks/cards/cards.css

```diff
@@ -609,6 +609,7 @@ main > .section > .default-content-wrapper {
     display: grid;
     grid-template-columns: repeat(auto-fill, minmax(40%, 1fr));
     gap: 40px 20px;
+    max-width: 63.25rem;
```

Comment:
the content should be center aligned. Refer: https://www.ingredion.com/na/en-us/solving-a-challenge/innovation-with-idea-labs/snacking-innovation-community/consumer-challenges.html